sprocs/users_select_or_insert.sql

```diff
@@ -4,6 +4,7 @@ CREATE FUNCTION
         IN name text,
         IN uin text,
         IN authn_provider_name text,
+        In institution_id bigint,
```

Comment:
Avoid needing to pass `null` everywhere. I'm also fine if you prefer to not have a default value so as to force to explicitly pass `null`.
```suggestion
        IN institution_id bigint DEFAULT NULL,
```

Reply:
Ah nice - I'll switch to using a default.

Reply:
Changed in 0b0624a.